Cargo.toml

```diff
@@ -47,8 +47,7 @@ blake2 = { version = "0.9", optional = true }
 rand = { version = "0.7", optional = true }
 sha2 = { version = "0.9", optional = true }
 x25519-dalek = { version = "1.1", optional = true }
-pqcrypto-kyber = { version = "0.6", optional = true }
-pqcrypto-traits = { version = "0.3", optional = true }
+oqs = { version = "0.3.0", optional = true }
```

Comment:
Please turn off the default features. Statically linking to OpenSSL is a headache on every platform, but especially broken on Windows as [noted in this issue](https://github.com/open-quantum-safe/liboqs-rust/issues/17).

Reply:
i guess you mean the default features in oqs? i'll look into it.

Reply:
right right. @BlackHoleFox okay i've disabled the default features in oqs.